apps/frontend/components/PrintHeader/PrintHeader.tsx

```diff
@@ -28,30 +28,33 @@ export const PrintHeader = ({ searchReportData }: IProps): JSX.Element => {
           </>
         ))}
       </Typography>
-      {searchReportData.region.province !== '' ? (
+      {searchReportData.region.province.length !== 0 ? (
         <Typography ml={3}>
           <FormattedMessage id="validation_search_params.province" />
           {' : '}
+          {/* TODO: How should we display things here? */}
```

Comment:
Can you explicit this todo?

Reply:
we used to show the selected province/district here, I now join the selected provinces with a comma. 
But this is only shown when we print the page, from the browser, not when we use the print icon
.Since this is more of a comment now than a todo now, i moved it and re phrased it